scripts/build/add-basic-constructor-to-react-component.js

```diff
@@ -0,0 +1,37 @@
+// this is to avoid the spread argument helper for react component constructors
+// https://runkit.com/mitchellhamilton/5b370eeaf5b34c0012874d92
+// without this, the constructor's arguments will be iterated over and put in an array
+module.exports = function(babel) {
+  const { types: t } = babel
+  let isReactComponent = t.buildMatchMemberExpression('React.Component')
+  let hasConstructor = body =>
+    body.length &&
+    body.some(node => {
+      return (
+        t.isClassMethod(node) &&
+        t.isIdentifier(node.key) &&
+        node.key.name === 'constructor'
+      )
+    })
+  return {
+    name: 'ast-transform', // not required
+    visitor: {
+      Class(path) {
+        if (
+          isReactComponent(path.node.superClass) &&
```

Comment:
not yet sure how & where this is used, maybe it doesnt matter in current context - but would be good to support `PureComponent` in `isReactComponent` function